src/core/server/models/comment/comment.ts

```diff
@@ -734,9 +734,6 @@ export async function updateCommentStatus(
       id,
       tenantID,
       "revisions.id": revisionID,
-      status: {
-        $ne: status,
-      },
```

Comment:
Is this being done for index reasons @wyattjoh? It causes issues when we want to "approve" a comment that has already been approved.

i.e
- Staff asks a question
- Is auto-approved cause they're staff
- Expert answers their question
- System tries to approve parent question cause it's an answered question
- This filter causes the `$ne: APPROVED` on that pre-approved comment to return a not found for this comment update
- Error occurs preventing the comment from being replied to/answered